src/foam/lib/formatter/AbstractFObjectFormatter.java

```diff
@@ -116,6 +118,32 @@ protected synchronized List getProperties(ClassInfo info) {
     return propertyMap_.get(of);
   }
 
+  protected synchronized List getDelta(FObject oldFObject, FObject newFObject) {
```

Comment:
Why synchronized?

Reply:
Ah, it initially didn't use getProperties itself. So this can now be removed.

Reply:
fixed.